examples/pytorch/BertNewsClassification/conda.yaml

```diff
@@ -13,6 +13,6 @@ dependencies:
   - sklearn
   - cloudpickle==1.6.0
   - boto3
-  - transformers
+  - transformers<4.0.0
```

Comment:
Avoid the latest version released today: https://pypi.org/project/transformers/4.0.0/

Reply:
What's the problem with the latest `transformers`? Is there a github issue tracking the problem?

Reply:
For posterity, I think that `!=` is generally better than `<` for examples / tests, assuming that the issue is corrected in a subsequent release.

Reply:
@dbczumar Thanks for the advice! I'll do so from the next time:)

I investigated this issue. In transformers 4.0.0, a model returns a dict:

```
outputs = model(**inputs)  # outputs is a dict.
```

So when we unpack an model output like below, the second key (`str`) will be assigned to `pooled_output` which is supposed to be a torch tensor.

```
_, pooled_output = self.bert_model(input_ids=input_ids, attention_mask=attention_mask)
```

Reply:
I'll fix this.

Reply:
Opened #3774